config/locales/en.yml

```diff
@@ -20,16 +20,17 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  layout:
```

Comment:
...got you, this is really a special case.

On Wed, Jan 21, 2015 at 2:47 PM, Joel Low <notifications@github.com> wrote:

> In config/locales/en.yml
> <https://github.com/Coursemology/coursemology2/pull/73#discussion_r23280930>
> :
>
> > @@ -20,16 +20,17 @@
> >  # available at http://guides.rubyonrails.org/i18n.html.
> >
> >  en:
> > -  layout:
>
> ...that's why I didn't use layouts, because this is meant to be shared
> among themes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/73/files#r23280930>.
>